packages/medusa/src/types/line-item.ts

```diff
@@ -0,0 +1,50 @@
+export type GenerateConfig = {
+  unit_price?: number
+  metadata?: object
+}
+
+export type createLineItemDto = {
```

Comment:
very nit: Should be capitalized and renamed to be consistent with other services: `CreateLineItemInput`